content/pt-br/README.md

```diff
@@ -189,20 +191,6 @@ Caso você precise de ajuda em algum momento ao contribuir, os [Embaixadores par
 | -------------------------- | -------------------------- | -------------------------- |
 | Sreeram Venkitesh                | @sreeram.venkitesh                      | @sreeram-venkitesh              |
 
-## Traduções do `README.md`
-
-| Idioma                    | Idioma                     |
-| ------------------------- | -------------------------- |
-| [Alemão](README-de.md)    | [Italiano](README-it.md)   |
-| [Chinês](README-zh.md)    | [Japonês](README-ja.md)    |
-| [Coreano](README-ko.md)   | [Polonês](README-pl.md)    |
-| [Espanhol](README-es.md)  | [Português](README-pt.md)  |
-| [Francês](README-fr.md)   | [Russo](README-ru.md)      |
-| [Hindi](README-hi.md)     | [Ucraniano](README-uk.md)  |
-| [Indonésio](README-id.md) | [Vietnamita](README-vi.md) |
-
-Você pode contatar os mantenedores da localização em Português no canal do Slack [#kubernetes-docs-pt](https://kubernetes.slack.com/messages/kubernetes-docs-pt).
```

Comment:
I wasn’t sure where to move this text. 

I moved it to the `Comunidade, discussão, contribuição e apoio` section, but if that’s incorrect, please feel free to adjust it. Alternatively, you can close this PR and create a new one.